frontend/api/index.js

```diff
@@ -111,14 +111,86 @@ app.get('/config/project-overrides', (req, res) => {
       value: envToBool('DISABLE_INVITE_LINKS', false),
     },
   ]
-  const output = values.map(getVariable).join('')
-
+  let output = values.map(getVariable).join('')
+  let dynatrace = ''
+  if (process.env.DYNATRACE_URL) {
```

Comment:
If DYNATRACE_URL env var is set, add the following snippet to the end of project-overrides:

Reply:
This is uber skank but not sure if there is an alternative?

Reply:
There isnt